recipes/sdl_ttf/all/conanfile.py

```diff
@@ -88,9 +88,14 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "SDL2_ttf.framework"))
 
     def package_info(self):
+        self.cpp_info.set_property("cmake_file_name", "SDL2_ttf")
+        self.cpp_info.set_property("cmake_target_name", "SDL2_ttf::SDL2_ttf")
+        self.cpp_info.set_property("pkg_config_name", "SDL2_ttf")
```

Comment:
bump `required_conan_version` to 1.43.0 please

Reply:
Thank you. I fixed it.